include/timing/timing.h

```diff
@@ -10,15 +10,49 @@
 #include <kernel.h>
 
 
+typedef uint64_t timing_t;
+
+void arch_timing_init(void);
+void arch_timing_start(void);
+void arch_timing_stop(void);
+timing_t arch_timing_counter_get(void);
+uint64_t arch_timing_cycles_get(volatile timing_t *const start,
+				volatile timing_t *const end);
+uint64_t arch_timing_freq_get(void);
```

Comment:
these prototypes need to be in an arch_interface header and not here, with complete documentation.

Reply:
Moved these functions to `sys/arch_interface.h`.